tests/browser_context_test.go

```diff
@@ -0,0 +1,82 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/dop251/goja"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserContextAddCookies(t *testing.T) {
+	opts := defaultLaunchOpts()
+	tb := newTestBrowser(t, withFileServer(), opts)
```

Comment:
The following is the same (the default options are used by default):

```suggestion
	tb := newTestBrowser(t, withFileServer())
```